lib/runner.js

```diff
@@ -65,6 +65,10 @@ optionChain(chainableMethods, function (opts, title, fn) {
 		throw new TypeError('Expected a function. Use `test.todo()` for tests without a function.');
 	}
 
+	if (typeof fn !== 'function') {
+		throw new TypeError('Expected a function. Use `test.todo()` for tests without a function.');
```

Comment:
You now have the exact same error defined twice. Put it in a variable and reuse it.